portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,28 @@
 <!DOCTYPE html>
 <html>
+<body bgcolor="#BBFFFF">
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <!--<body>-->
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Korhan Citlak's Portfolio</h1>
+      <p>I am a sophomore studying computer science at Binghamton University. <br>
+        I grew up with technology all around me and it helped me deal with my disability.
+        That is why I want to make software that will positively impact people's lives: to give back.  </p> 
+      <br>
+      <p> On a less serious not, here is a picture of my cute cat, a maine coon, named Prince! <br> </p>
```

Comment:
Whoops, small typo here `not` --> `note`. I would also recommend putting the `<br>` outside of the `<p>...</p>` tags if you're just going for some extra space in between the text and the image. There's no hard rule about it, but I think in general it's easier to keep track of what tags you've got if they're as un-nested as possible.